ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -124,6 +124,7 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
     });
     // since the request resolved trigger a page change and return an error from the associations endpoint
     await click(pagination.next);
+    await waitFor('[data-test-empty-state-title]', { timeout: 5000 });
```

Comment:
Is there any other option than this? I'm concerned this could be flaky given that it is waiting an arbitrary amount of time. Since we're doing a transition the test should wait since `click` calls `settled` internally but guessing that's not the case?

Reply:
I'll go ahead and remove it and if we still have issues in the data branch I can revisit. But I agree.